Entities/Common/Emotes/EmoteHotkeys.as

```diff
@@ -34,6 +44,16 @@ void onInit(CBlob@ this)
 	emote_8 = read_emote(cfg, "emote_8", Emotes::disappoint);
 	emote_9 = read_emote(cfg, "emote_9", Emotes::ladder);
 
+    emote_10 = read_emote(cfg, "emote_10", Emotes::flex);
```

Comment:
This line still looks off with indentation. Editors let you reindent whole files. You don't have to do it manually.

Reply:
Oops, I totally forgot that! I'll give it a try.